### PR TITLE
Aerogear services update

### PR DESCRIPTION
**Pull request by maleck13**

### CODE REVIEW

File: services/keycloak/index.html [hunk 1]
```diff
@@ -1,5 +1,45 @@
 ---
 layout: index
-title: Aerogear Keycloak
+title: Identity Management
 section: mobile-services
 ---
+
+<div class="row">
+    <div class="col-md-2">
+        <img style="width: 100px; height: 100px;" src="/img/keycloak.png" />
```
Comment (dimitraz) [inline services/keycloak/index.html]: Personally I would remove images and icons from the titles. Just my preference though

File: services/keycloak/index.html [hunk 2]
```diff
@@ -1,5 +1,45 @@
...
+    </p>
+</div>
+<div class="row">
+    <h3>Documentation</h3>
+    <a target="_blank" href="https://docs.aerogear.org" class="btn btn-primary">
+        <i class="fa fa-book"></i> Guides! Docs! Learn more</a>
+</div>
+<div class="row">
+    <h3>Github</h3>
+    <a target="_blank" href="https://github.com/aerogearcatalog/keycloak-apb/" class="btn btn-primary btn-sm">
```
Comment (dimitraz) [inline services/keycloak/index.html]: Can the `btn-sm` be removed from the github icons? Might look better when it's the same size as the docs button

File: services/mobile-ci-cd/index.html [hunk 3]
```diff
@@ -1,5 +1,38 @@
...
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas magna ante, cursus sit amet accumsan at, sodales blandit
+        ante. Proin cursus sagittis vehicula. Nulla mattis id erat non sagittis. Nullam ut ligula venenatis, ultrices nulla
+        a, convallis sem. Interdum et malesuada fames ac ante ipsum primis in faucibus. Vestibulum tincidunt, erat quis tincidunt
+        bibendum, est sapien viverra erat, et dictum felis dolor vel nulla. Sed hendrerit tincidunt ligula id mollis. Nam
+        iaculis lectus risus, sed suscipit dui tempus vitae. Pellentesque nec ultrices risus, sit amet vehicula nisi. </p>
+</div>
+<div class="row">
+    <h3>Getting Started</h3>
+    <p>This guide will show you how to set up the metrics service and use it to gather application and service metrics.
+        <a target="_blank" href="https://github.com/aerogearcatalog/metrics-apb/blob/master/docs/modules/ROOT/pages/index.adoc">Geting started with metrics for mobile</a>
```
Comment (dimitraz) [inline services/mobile-ci-cd/index.html]: Is there a link to the ci cd docs

File: services/mobile-ci-cd/index.html [hunk 4]
```diff
@@ -1,5 +1,38 @@
...
+    </p>
+</div>
+<div class="row">
+    <h3>Documentation</h3>
+    <a target="_blank" href="https://docs.aerogear.org" class="btn btn-primary"><i class="fa fa-book"></i> Guides! Docs! Learn more</a>
+</div>
+<div class="row">
+    <h3>Github</h3>
+    <a target="_blank" href="https://github.com/aerogearcatalog/metrics-apb/" class="btn btn-primary btn-sm"><i class="fa fa-github-alt"></i> View the Source</a>
```
Comment (dimitraz) [inline services/mobile-ci-cd/index.html]: Same with link here for ci cd

File: services/mobile-ci-cd/index.html (hunk 3 above)
Comment (maleck13) [reply services/mobile-ci-cd/index.html]: woops good spot!

File: services/keycloak/index.html (hunk 1 above)
Comment (maleck13) [reply services/keycloak/index.html]: The acceptance crit asked for the icon. Will leave it for now and others might choose to move it somewhere else

Comment (dimitraz) [reply services/keycloak/index.html]: Ok

**Comment by maleck13** [comment]
ping @pb82 this is the pr that will merge shortly

**Comment by pb82** [comment]
@maleck13 Ok, we will update the links here with the proper ones, once the docs are in place.

File: broker.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
+{
```
Comment (dimitraz) [inline broker.json]: Was this added in accidentally?

File: services/mobile-ci-cd/index.html [hunk 6]
```diff
@@ -1,5 +1,41 @@
...
+        bibendum, est sapien viverra erat, et dictum felis dolor vel nulla. Sed hendrerit tincidunt ligula id mollis. Nam
+        iaculis lectus risus, sed suscipit dui tempus vitae. Pellentesque nec ultrices risus, sit amet vehicula nisi. </p>
+</div>
+<div class="row">
+    <h3>Getting Started</h3>
+    <p>This guide will show you how to set up the CI CD service on OpenShift.
+        <a target="_blank" href="https://github.com/aerogear/mobile-core/blob/master/docs/walkthroughs/provision-mobile-ci-cd.adoc">Provisioning the mobile CI CD service</a>
+    </p>
+    <p>
+        This guide will show how to build your mobile client binaries using the mobile CI CD service through OpenShift <a target="_blank" href=" https://github.com/aerogear/mobile-core/blob/master/docs/walkthroughs/create-mobile-client-build.adoc" >Geting started with the mobile CI CD service</a>
```
Comment (JameelB) [inline services/mobile-ci-cd/index.html]: Typo on `Getting started..` same for all the other services

**Comment by maleck13** [comment]
The overview will be filled in by docs

**Pull request merged**